src/ETISS.cpp

```diff
@@ -744,6 +744,7 @@ void etiss_initialize(const std::vector<std::string>& args, bool forced = false)
             ("etiss.output_path_prefix", po::value<std::string>(), "Path prefix to use when writing output files.")
             ("etiss.loglevel", po::value<int>(), "Verbosity of logging output.")
             ("jit.gcc.cleanup", po::value<bool>(), "Cleans up temporary files in GCCJIT. ")
+            ("jit.gcc.opt_level", po::value<std::string>(), "GCCJIT optimization level (0/1/2/3/s/fast). ")
```

Comment:
Please add the default option `fast` to the helper string, e.g., "(0/1/2/3/s/fast=default)"